### ISSUE TITLE
Inappropriate Armor Check Penalties 

### ISSUE THREAD

**Issue by Legion50** [GrognardsFromHell/TemplePlus#803]
The following armors have inappropriate armor check penalties being applied when they should be 0, resulting in penalties to Hide, Move Silently, Sleight of Hand, and Tumble.

- Darkwood Buckler
- Large Darkwood Shield
- Large Mithral Shield
- Mithral Shield
- Wooden Elvish Shield

This list may not be exhaustive, I only tested equipment that my party would be interested in using at the time.

**Comment by dolio** [comment]
The items don't appear to have any net penalty. The details show a -1 for the shield, but then +1 for the shield being masterwork, which is net 0. I wouldn't be surprised if the penalty is there just to prevent the masterwork part from making it a net increase to your skills (because I think the masterwork condition just provides a bonus to the relevant things).

I could change the way things are calculated to look different, but I don't think it would make any difference in the end results.

**Comment by Legion50** [comment]
Hmm... I was certain something was off. Let me double-check in a bit.

**Comment by dolio** [comment]
Yeah, let me know if you see something where the two don't offset each other. But I'm pretty sure the purpose is what I described. Most large shields have a -2 penalty, and I think the -1 penalty on the ones you mention is there because the masterwork condition gives a flat +1 bonus to all the things armor check penalties apply to. I'm not sure why they decided to keep track of it that way, but they did.

**Comment by Legion50** [comment]
Yep, I think you're right. I've confirmed that behavior with the Mithral Shirt and Wooden Elvish Shield already in my party. That's so... goofy, but I guess it works for its purpose lol. Thanks.

**Comment by dolio** [comment]
I might change up the way this works, because I noticed that if you wear masterwork leather, it actually gives you a net +1 to various skills, because leather has no penalty.

**Comment by doug1234** [comment]
> I might change up the way this works, because I noticed that if you wear masterwork leather, it actually gives you a net +1 to various skills, because leather has no penalty.

In that case it would definitely be a good idea to update it.  Thanks!

**Comment by dolio** [comment]
Been working on this, and just thought I'd mention how convoluted the existing situation was.

So, actually, what I noticed was that the pre-canned +3 leather armor gives you a net +1 to various skills, this is because the enhancement conditions have most of the same behavior as masterwork, because the protos don't bother to put both conditions on items (this also means casting Magic Weapon on something turns it into a masterwork item, which I'll eventually fix).

But, actually, someone thought of the problem for the Masterwork Leather proto. There are some -1s to offset stuff that the masterwork condition buffs.

But, they didn't just give the masterwork leather a base -1 armor check penalty. What they did was add a bunch of circumstance penalties to skills. So it won't actually offset everything that the mastwerwork condition buffs.

Also, the proto says there's a circumstance penalty to "skill_use_magic_device". But I'm pretty sure that's applying to tumble. 😛

**Comment by Legion50** [comment]
Closing this, fixed as of 1.0.98. Thanks!
